canonical-websites/www.ubuntu.com
# remove local help files for tutorials

## Done 

* updated links to use new tutorials
* removed pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the /download section, including thank-you pages to confirm they are all moved and external links correct

## Issue / Card

Fixes #1640 - /download/server/install-ubuntu-server/ to VBT Status 
Fixes #1637 - /download/how-to-verify/ to VBT Status 
Fixes #1636 - /download/desktop/upgrade to VBT Status 
Fixes #1635 - /download/desktop/try-ubuntu-before-you-install/ to VBT Status 
Fixes #1633 - /download/desktop/install-ubuntu-desktop/ to VBT Status 
Fixes #1632 - /download/desktop/create-a-usb-stick-on-windows/ to VBT Status 
Fixes #1631 - /download/desktop/create-a-usb-stick-on-ubuntu/ to VBT Status 
Fixes #1630 - /download/desktop/create-a-usb-stick-on-macos/ to VBT Status 
Fixes #1628 - /download/desktop/burn-a-dvd-on-windows/ to VBT Status 
Fixes #1627 - /download/desktop/burn-a-dvd-on-ubuntu/ to VBT Status 
Fixes #1626 - /download/desktop/burn-a-dvd-on-macos/ to VBT Status
